### ISSUE TITLE
Score missing from the chart

### ISSUE THREAD

**Issue by simonemainardi** [ntop/ntopng#3802]
Given the following alerts

![image](https://user-images.githubusercontent.com/3397663/79755582-a5f5ec80-8319-11ea-8f00-e6c3cd1d9a20.png)

I would have expected to see scores > 0 around  14:41:02 in the timeseries.

However, both of them are empty around the range of interest:

![image](https://user-images.githubusercontent.com/3397663/79755676-caea5f80-8319-11ea-8959-96fb814cd28c.png)

![image](https://user-images.githubusercontent.com/3397663/79755741-df2e5c80-8319-11ea-8931-383e6279dec3.png)

**Comment by emanuele-f** [comment]
I've verified that there is an issue with the first point (the first gauge value is only kept into the metadata), so this is probably solved after #3822 is fixed.

**Comment by lucaderi** [comment]
Note that scores as missing from the Interface chart (shown only on the host chart). This need to be added

**Comment by emanuele-f** [comment]
Interface score chart added

**Comment by lucaderi** [comment]
Is it possible to drill down hosts according to score? I.e. top hosts according to score? I don't think so. So let me open a new ticket. I have open #3843 for this
